### PR TITLE
Update the content of Package.md and README.md file & Replace icon with high resolution image

### PR DESCRIPTION
**Pull request by MadhukaHarith92**
## Purpose
https://github.com/ballerina-platform/ballerina-standard-library/issues/2657
## Checklist
- [x] Linked to an issue
- [ ] Updated the changelog
- [ ] Added tests

### CODE REVIEW

File: ballerina/Module.md [hunk 1]
```diff
@@ -21,5 +21,3 @@ Uses a pseudo-random number generator to generate the UUID. Every bit of the str
 Similar to Version 3 but uses SHA-1 instead of MD5.
 
 Other operations include validating a given UUID string and getting the version of a UUID string.
-
-For information on the operations, which you can perform with the `uuid` module, see the below **Functions**.
```
Comment (daneshk) [inline ballerina/Module.md:25]: Shall we keep these lines in module.md

File: README.md [hunk 2]
```diff
@@ -7,13 +7,27 @@ Ballerina Uuid Package
...
-The `uuid` package is one of the standard library packages of the<a target="_blank" href="https://ballerina.io/"> Ballerina</a> language.
+This library provides APIs to generate and inspect UUIDs (Universally Unique Identifiers).
 
-This package provides APIs to generate and inspect UUIDs (Universally unique identifier).
+The UUIDs are generated based on the [RFC 4122](https://www.rfc-editor.org/rfc/rfc4122.html) standard. This module supports generating 4 versions of UUIDs.
 
-The UUIDs are generated based on the [RFC 4122](https://www.rfc-editor.org/rfc/rfc4122.html) standard. Other operations include validating a given UUID string and getting the version of a UUID string.
+### Version 1
 
-For example demonstrations of the usage, go to [Ballerina By Examples](https://ballerina.io/swan-lake/learn/by-example/).
```
Comment (daneshk) [inline README.md:16]: We need to add this line in useful links section

File: ballerina/Module.md (hunk 1 above)
Comment (MadhukaHarith92) [reply ballerina/Module.md]: Update in [dcd7453](https://github.com/ballerina-platform/module-ballerina-uuid/pull/362/commits/dcd74535d86bec7c7f1558727e4c328a69ea70d1)

File: README.md (hunk 2 above)
Comment (MadhukaHarith92) [reply README.md:16]: Updated in [dcd7453](https://github.com/ballerina-platform/module-ballerina-uuid/pull/362/commits/dcd74535d86bec7c7f1558727e4c328a69ea70d1)

**Review by daneshk — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -1,4 +1,4 @@
-Ballerina Uuid Package
+Ballerina Uuid Library
```
Comment (BuddhiWathsala) [inline README.md:1]: I think this should be UUID, right? @MadhukaHarith92

Comment (MadhukaHarith92) [reply README.md:1]: Fixed in https://github.com/ballerina-platform/module-ballerina-uuid/pull/363

**Pull request merged**